From 97b494847ca65a442ec8a2529c7909363067cd80 Mon Sep 17 00:00:00 2001
From: Gao Pan <pandy.gao@nxp.com>
Date: Wed, 24 Jan 2018 14:00:02 +0800
Subject: [PATCH 3250/5242] MLK-17453 arm64: dts: increase i2c pad drive
 strength

commit  2087a89cb3f2bf42c075b12c216b2d138918f328 from
https://source.codeaurora.org/external/imx/linux-imx.git

Too many i2c slaves are conencted to imx8qm i2c0 and
imx8qxp lpi2c1. So i2c pad drive strength should be
increased. Otherwise, there will be i2c probe error.

Reported-by: Andy Tian <yang.tian@nxp.com>
Tested-by: Andy Tian <yang.tian@nxp.com>
Signed-off-by: Gao Pan <pandy.gao@nxp.com>
Acked-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts  |    4 ++--
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index a412d19..fb2fa75 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -340,8 +340,8 @@
 
 		pinctrl_i2c0: i2c0grp {
 			fsl,pins = <
-				SC_P_HDMI_TX0_TS_SCL_DMA_I2C0_SCL	0x0600004c
-				SC_P_HDMI_TX0_TS_SDA_DMA_I2C0_SDA	0x0600004c
+				SC_P_HDMI_TX0_TS_SCL_DMA_I2C0_SCL	0x06000021
+				SC_P_HDMI_TX0_TS_SDA_DMA_I2C0_SDA	0x06000021
 			>;
 		};
 
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index 205f2e5..4393a42 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -327,8 +327,8 @@
 
 		pinctrl_lpi2c1: lpi1cgrp {
 			fsl,pins = <
-				SC_P_USB_SS3_TC1_ADMA_I2C1_SCL	0x06000020
-				SC_P_USB_SS3_TC3_ADMA_I2C1_SDA	0x06000020
+				SC_P_USB_SS3_TC1_ADMA_I2C1_SCL	0x06000021
+				SC_P_USB_SS3_TC3_ADMA_I2C1_SDA	0x06000021
 			>;
 		};
 
-- 
1.7.9.5

